cctk/orca_file.py

```diff
@@ -197,20 +185,26 @@ def read_file(cls, filename):
                 properties[-1]["frequencies"] = sorted(parse.read_freqs(lines, successful_freq))
 
                 enthalpies = lines.find_parameter("Total Enthalpy", expected_length=5, which_field=3)
-                # if len(enthalpies) > 0:
-                #  for jobs that don't converge, there are no enthalpies, so this errors out
-                #  need to add a "try, else", so that it returns OrcaFile.read_file(path) returns an orca file and prints a warning.
-                properties[-1]["enthalpy"] = enthalpies[-1]
+                try:
+                    properties[-1]["enthalpy"] = enthalpies[-1]
+                except Exception as e:
+                    print(f"error finding enthalpy for {filename}")
```

Comment:
When would an enthalpy not be found?  Are there cases where you would have more than one enthalpy?